web/ui/dashboard/src/stories/input/Input.stories.ts

```diff
@@ -0,0 +1,93 @@
+import { Story, Meta } from '@storybook/angular/types-6-0';
+import { moduleMetadata } from '@storybook/angular';
+import { InputComponent } from '../../app/components/input/input.component';
+import { FormControl, FormGroup, ReactiveFormsModule } from '@angular/forms';
+
+export default {
+	title: 'Example/Input',
+	component: InputComponent,
+	decorators: [
+		moduleMetadata({
+			imports: [ReactiveFormsModule]
+		})
+	],
+	argTypes: {
+		label: {
+			control: { type: 'text' }
+		},
+		type: {
+			options: ['text', 'password', 'number', 'url', 'email'],
+			control: { type: 'text' },
+			defaultValue: 'text'
+		},
+		formControlName: {
+			control: { type: 'text' }
+		},
+		errorMessage: {
+			control: { type: 'text' }
+		},
+		placeholder: {
+			control: { type: 'text' }
+		},
+		required: {
+			control: { type: 'boolean' }
+		},
+		readonly: {
+			control: { type: 'boolean' }
+		}
+	},
+	parameters: {
+		actions: {
+			handles: ['click']
```

Comment:
Is click necessary here?
Instead, it should be frequent input events like focus, keyup, e.t.c.